### DISCUSSION TITLE
0.12-rc.2

### DISCUSSION THREAD

**Discussion by JohnsonRan** [DUpdateSystem/UpgradeAll#84] [General]
1. Update dependencies
2. Add German language
3. Optimize discover dialog ui

**Full Changelog**: https://github.com/DUpdateSystem/UpgradeAll/compare/0.12-rc.1...0.12-rc.2

<hr /><em>This discussion was created from the release <a href='https://github.com/DUpdateSystem/UpgradeAll/releases/tag/0.12-rc.2'>0.12-rc.2</a>.</em>

**Comment by MadTooth** [comment]
Very Good Very Nice
